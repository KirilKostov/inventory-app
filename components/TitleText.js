import React from "react";
import { Text, StyleSheet } from "react-native";

export const TitleText = (props) => {
  return (
    <Text style={{ ...styles.titleText, ...props.style }}>
      {props.children}
    </Text>
  );
};

const styles = StyleSheet.create({
  titleText: {
    fontFamily: "gibson-semibold",
    fontSize: 22,
  },
});
