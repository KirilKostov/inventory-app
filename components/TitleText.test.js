import React from "react";
import renderer from "react-test-renderer";

import { TitleText } from "./TitleText";

describe("<TitleText />", () => {
  it("renders correctly", () => {
    const testRenderer = renderer
      .create(<TitleText>Fantastic title text</TitleText>)
      .toJSON();
    expect(testRenderer).toMatchSnapshot();
  });
});
