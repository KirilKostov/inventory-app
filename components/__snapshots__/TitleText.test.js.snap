// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TitleText /> renders correctly 1`] = `
<Text
  style={
    Object {
      "fontFamily": "gibson-semibold",
      "fontSize": 22,
    }
  }
>
  Fantastic title text
</Text>
`;
